agr_mvs/turno.py

```diff
@@ -0,0 +1,118 @@
+
+    '''
+    Para el calculo de λ debemos de tener en cuenta que lo hacemos por hora,
+    con fin de simplificar el cálculo, vamos a realizar una división entre el numero de tickets 
+    que tenemos en el turno y las horas del turno. (Representamos cada cliente por un ticket sin valorare el importe del ticket)
+    El resultado lo vamos a dividir entre 60 para trabajar con minutos en los posteriores cálculos.
+    '''
+    def calculo_clientes_unidad_tiempo(self):
+        return ((len(self.tickets)/HORAS_TURNO)/60.0)
+    
+    '''
+    Calculo variables elementales para el calculo de las variables estadísticas, 0 contiene µ, 1 contiene λ y 2 contiene S. 
+    '''
+    def calculo_variables_elementales(self):
+        variables_elementales = []
+        variables_elementales.append(self.calculo_maximo_clientes_atendidos())
+        variables_elementales.append(self.calculo_clientes_unidad_tiempo())
+        variables_elementales.append(self.S)   
+        return variables_elementales     
+    
+    '''
+    Mediante esta funcion vamos a poder calcular la saturacion de la tienda.
+    '''
+    def calculo_saturacion(self):
+        variables = self.calculo_variables_elementales()
+        saturacion = variables[1]/(variables[0]*variables[2])
+        return saturacion
+    
+    
+    '''
+    Calculo del numero promedio de clientes en la cola: Lq = p / Ls (Siendo Ls el numero promedio de clientes que entran al sistema)
+    Ls = λ / (µ - λ)
+    '''
+    def calculo_promedio_clientes_cola(self):
+        saturacion = self.calculo_saturacion()
+        variables = self.calculo_variables_elementales()
+        promedio_clientes = saturacion * (variables[1] / (variables[0] - variables[1]))
+        return promedio_clientes
+
+    '''
+    Calculo del tiempo promedio de espera de un cliente en la cola Wq = Lq/λ ( Vamos a expresar λ en minutos para que sea mas orientativo el resultado) 
+    '''
+    
+    def calculo_tiempo_espera_cola(self):
+        variables = self.calculo_variables_elementales()
+        promedio_clientes = self.calculo_promedio_clientes_cola()
+        tiempo_espera_cola = (promedio_clientes / variables[1])
+        return tiempo_espera_cola
+
+    '''
+    Mediante este calculo vamos a poder obtener la probabilidad de que un cliente tenga que esperar en la cola mas de 30 minutos
+    que es el tiempo a partir del cual se considera que podemos perder clientes. la formula es: p * e^(-µ*(1-p)*t). En este caso
+    vamos a expresar el parámetro µ en minutos para que el resultado que arroje se base en minutos y no en horas. 
+    '''
+    def calculo_probabilidad_espera_cola(self,minutos):
+        saturacion = self.calculo_saturacion()
+        variables = self.calculo_variables_elementales()
+        exp = ((-variables[0]) * (1 - saturacion) * minutos)
+        probabilidad_cola = saturacion * math.exp(exp)
+        return probabilidad_cola
+    
+    def calculo_variables_estadísticas(self):
```

Comment:
Una función nunca debe realizar ninguna operación de entrada salida: *separation of concerns* significa que una función debe hacer una cosa, y solo una, y el identificador debe decir claramente lo que hace. "Calculo" no incluye "imprimir en salida estándar". La salida estándar, además, no se puede usar de ninguna forma por el cliente de la clase.
En todo caso, crear funcionalidad que no pida ningún issue *siempre* es un error.

Reply:
Se ha eliminado esa funcion ya que su resultado se puede obtener con otras funciones